iOS/README.md

```diff
@@ -0,0 +1,8 @@
+### Setup instructions for AugmentedFacesExample app.
+
+From the /MMIWG2S/iOS directory in terminal:
+1. Run `$ pod install`  
```

Comment:
Could be good to include some instructions for installing cocoapods.

Reply:
Thanks! I forget about that since I've had it installed since the beginning of time haha.